modules/apps/journal/journal-service/src/main/java/com/liferay/journal/util/impl/JournalContentImpl.java

```diff
@@ -217,6 +223,10 @@ public JournalArticleDisplay getDisplay(
 			layoutSetId = layoutSet.getLayoutSetId();
 
 			secure = themeDisplay.isSecure();
+
+			if (!lifecycleRender) {
+				lifecycleRender = themeDisplay.isLifecycleRender();
```

Comment:
@jcampoy do you really mean to use themeDisplay.isLifecycleRender() to overwrtie portletRequestModel.getLifecycle()? Assuming they both not null and portletRequestModel.getLifecycle() is not render phase.
Or you are trying to say when portletRequestModel is null, use themeDisplay.isLifecycleRender() as fallback?
Your code is doing overwrite, but your first commit's message seems to say it is a fallback.

Please clarify what's the intention.

Reply:
Good catch, I would do it in a conservative way, having into account that **portletRequestModel** is null in this condition to prevent a possible regression, in spite of the **themeDisplay** had preference over the previous **xmlRequest** before the refactoring of https://issues.liferay.com/browse/LPS-45558 , see: https://github.com/liferay/liferay-portal-ee/blob/ee-6.2.x/portal-impl/src/com/liferay/portlet/journalcontent/util/JournalContentImpl.java#L308-L322

But really I do not know why we have into account the lifecycle for this util, could you let me know a case in where this cached value need to be discarded? 

Thanks!

PD: I am going to add a commit to check the _portletRequestModel_

Reply:
see https://github.com/shuyangzhou/liferay-portal/pull/7443/commits/82ecb9ca18eaac2888d5ba3e92baf84b6885025e